www/hoan01/ukol-3/task.css

```diff
@@ -0,0 +1,124 @@
+body {
+  font-family: Sans-Serif;
+}
+
+#colorName {
+	color:red;
+}
+
+#colorRGB {
+	color:rgb(0,255,0);
+}
+
+#colorHex {
+	color:#0000ff;
+	font-weight: bold;
+}
+
+#text-18 {
+	font-size: 18px;
+}
+
+#text-24 {
+	font-size: 24px;
+	font-style: italic;
+}
+
+#textWhite {
+	color:white;
+	background-color: brown;
+}
+
+#par-0 {
+	text-align: center;
+	line-height: 32px;
+}
+
+#par-1 {
+	display: flex;
+    justify-content: flex-start;
+    text-transform: uppercase;
+}
+
+#box-0 {
+	width: 400px;
+	height: 300px;
+	border: 1px solid purple;
+	padding: 30px 30px 30px 30px;
+}
+
+#box-1 {
+	width: 500px;
+	height: 200px;
+	background-color: red;
+	border: 1px solid purple;
+	margin: 100px 100px 100px 100px;
+}
+
+#textHide {
+	color:transparent;
+}
+
+#box-2 {
+	width: 100%;
```

Comment:
chybi box-sizing: border-box u vsech boxu, proto se stalo neco necekaneho u tohohle boxu: jeho sirka ma byt 100% stranky, ale ve skutecnosti je to trochu vetsi, je to protoze ma jeste padding a border, a ty se nezapocitavaji do zakladni sirky, chceme ale aby se zapocitavaly do width: 100%.